benchmarks/shelley3pools/benchmark.sh

```diff
@@ -47,24 +48,25 @@ done
 # 2 run rt-view
 tmux select-window -t :0
 tmux new-window -n RTview \
-             "${TMUX_ENV_PASSTHROUGH[*]} ./run-rt-view.sh; $SHELL"
+             "${TMUX_ENV_PASSTHROUGH[*]} bash ./run-rt-view.sh; $SHELL"
```

Comment:
Why do we need explicit `bash` here? If the script is executable, it can be launched without `bash`, just with `./`, right?

Reply:
Not quite -- people have different `$SHELL` values, and here we are trying to make it work for a person with non-bash.